test/test_models.py

```diff
@@ -364,7 +376,8 @@ def _check_input_backprop(model, inputs):
     "s3d": {
         "input_shape": (1, 3, 16, 224, 224),
     },
-    "googlenet": {"init_weights": True},
+    "regnet_y_128gf": {"weight_name": "IMAGENET1K_SWAG_LINEAR_V1"},
```

Comment:
Could we just get the actual weights from the model name, using the helpers from https://pytorch.org/vision/main/models.html#model-registration-mechanism ?

Reply:
We can, I actually use the helper to get the actual weight in [here](https://github.com/pytorch/vision/pull/7130/files#diff-cc89e674c41fe4051f693033089d1dd57f1065293c00ebd01ff1e8e92af2ff58R708).

I think I prefer this design where we dont need to specify the `weight_enum` for the `weight_name` (since it can be retrieve from the `model_name`). Also, it is easier to say that the default value that we use is `IMAGENET1K_V1` for the test.